GliaWidgets/Theme/Alert/VisitorCodeAlertConfiguration.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+public struct VisitorCodeAlertConfiguration {
```

Comment:
Seems like you don't use this configuration

Reply:
fixed [fc50a16](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/fc50a168934e072d2a4b217c22bcbdc235a31d95)